precompiles/ArbOwner.go

```diff
@@ -69,6 +69,9 @@ func (con ArbOwner) SetL2BaseFee(c ctx, evm mech, priceInWei huge) error {
 
 // SetMinimumL2BaseFee sets the minimum base fee needed for a transaction to succeed
 func (con ArbOwner) SetMinimumL2BaseFee(c ctx, evm mech, priceInWei huge) error {
+	if c.txProcessor.MsgIsNonMutating() && priceInWei.Sign() == 0 {
```

Comment:
the idea is to only fail eth_call / gas estimation and leave normal operation unchanged?

Reply:
Yep. It should be fine to change normal operation too, but I think let's leave that for the next ArbOS version.